Client/src/stores/user.ts

```diff
@@ -3,7 +3,7 @@ import { ActiveUser, ActiveUserModel } from '../models/user.model';
 import { useApi } from '../composables/useApi';
 
 export const userStore = reactive({
-  user: ref<ActiveUser>(new ActiveUserModel({})),
+  user: ref<ActiveUser>(),
```

Comment:
Wat is de reden dat je er niks in stopt? Met die class erin krijg je nooit verrassingen dat een property `undefined` is 😉